.gitignore

```diff
@@ -1,23 +1,11 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
-
-# production
-/build
-
-# misc
-.DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+*.pyc
+*.DS_Store
+node_modules
+.mypy_cache
+.pytest_cache
+.coverage*
+
+# vim temp files
+*~
+*.swo
+*.swp
```

Comment:
these should go in a personal `.gitignore`. Otherwise we have to include all the other patterns (emacs, etc.)